pandas/tests/base/test_constructors.py

```diff
@@ -13,6 +13,19 @@
 from pandas.core.base import NoNewAttributesMixin, PandasObject
 
 
+@pytest.fixture(
```

Comment:
@jorisvandenbossche if you hadn’t merged this so quickly i would have objected here

creating this fixture for a single use is not that great

we typically also call this klass and don’t much care about dataframe-dict here